h5c/vic/src/vic-webapp/src/app/vch-view/vch-view.component.ts

```diff
@@ -225,6 +231,50 @@ export class VicVchViewComponent implements OnInit, OnDestroy {
     this.vmViewService.getContainersData({}); // TODO: filter containers for the vch
   }
 
+   /**
+   * Download Certificate for VCH
+   */
+  downloadVchCert(vch: VirtualContainerHost) {
+    this.isDgLoading = true;
```

Comment:
Does this function actually have an impact on the datagrid? It doesn't seem like it should; we write the hidden anchor to the end of the body.

Reply:
Yes, isDgLoading is something we check on the grid. So it would launch the spinner over the grid while the download happens. But has nothing to do with the anchor itself.